SCOPES.md

```diff
@@ -0,0 +1,38 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:snapcraft`
+  * Used to tell which Google Play store a given apk should update.
+
+# Scopes
+* `{scope_prefix}:firefox:beta`
+  * Uploads to the *beta* snap track.
+    * Uses branding "Firefox Beta"
+  * **Conflicts**: with any other `{scope_prefix}:firefox:*`
+  * **Branch Restrictions**:
+    * `beta`
+
+* `{scope_prefix}:firefox:candidate`
+  * Uploads to the *candidate* snap track.
+    * Uses branding "Firefox"
+  * **Conflicts**: with any other `{scope_prefix}:firefox:*`
+  * **Branch Restrictions**:
+    * `release`
+
+* `{scope_prefix}:firefox:esr`
+  * Uploads to the ???.
+  * **Does not seem to be used**
+  * **Conflicts**: with any other `{scope_prefix}:firefox:*`
+  * **Branch Restrictions**: `None`
```

Comment:
I should restrict this scope to esr60.

Reply:
@JohanLorenzo reminder to do a PR for scriptworker to do that restriction.